ehrapy/preprocessing/_normalization.py

```diff
@@ -50,17 +68,19 @@ def scale_norm(adata: AnnData, vars: str | Sequence[str] | None = None, copy: bo
     var_idx = _get_column_indices(adata, vars)
     var_values = np.take(adata.X, var_idx, axis=1)
 
-    var_values = scale(var_values, **kwargs)
+    var_values = _scale_func_batch(scale, var_values, adata, batch_key, **kwargs)
 
     set_numeric_vars(adata, var_values, vars)
 
     _record_norm(adata, vars, "scale")
 
+    logg.debug("Scaling normalization was applied on `X`.")
```

Comment:
```suggestion
```

I removed all debug statements for now. We may re-introduce them at some point, but then with more rules and a proper design. We don't need them for now and they clutter the code.